### ISSUE TITLE
dbus-tokio 0.7 fails to build

### ISSUE THREAD

**Issue by birkenfeld** [diwic/dbus-rs#307]
Using Rust 1.49.
```
$ cargo new --bin foo
     Created binary (application) `foo` package
$ cd foo
$ cargo add dbus-tokio
    Updating 'https://github.com/rust-lang/crates.io-index' index
      Adding dbus-tokio v0.7.0 to dependencies
$ cargo +stable build
   Compiling proc-macro2 v1.0.24
   Compiling unicode-xid v0.2.1
   Compiling syn v1.0.57
   Compiling libc v0.2.81
   Compiling proc-macro-hack v0.5.19
   Compiling proc-macro-nested v0.1.6
   Compiling memchr v2.3.4
   Compiling pkg-config v0.3.19
   Compiling log v0.4.11
   Compiling futures-core v0.3.8
   Compiling cfg-if v0.1.10
   Compiling once_cell v1.5.2
   Compiling autocfg v1.0.1
   Compiling pin-utils v0.1.0
   Compiling futures-io v0.3.8
   Compiling slab v0.4.2
   Compiling pin-project-lite v0.2.1
   Compiling futures-channel v0.3.8
   Compiling futures-task v0.3.8
   Compiling tokio v1.0.1
   Compiling libdbus-sys v0.2.1
   Compiling quote v1.0.8
   Compiling mio v0.7.7
   Compiling pin-project-internal v1.0.3
   Compiling futures-macro v0.3.8
   Compiling pin-project v1.0.3
   Compiling futures-util v0.3.8
   Compiling dbus v0.9.1
   Compiling dbus-tokio v0.7.0
error[E0432]: unresolved import `dbus_tree`
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:335:9
    |
335 |     use dbus_tree::Factory;
    |         ^^^^^^^^^ use of undeclared crate or module `dbus_tree`

error[E0433]: failed to resolve: could not find `test` in `tokio`
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:314:10
    |
314 | #[tokio::test]
    |          ^^^^ could not find `test` in `tokio`

error[E0433]: failed to resolve: could not find `test` in `tokio`
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:332:10
    |
332 | #[tokio::test]
    |          ^^^^ could not find `test` in `tokio`

error[E0425]: cannot find function `spawn` in crate `tokio`
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:319:12
    |
319 |     tokio::spawn(async move { panic!(ress.await);});
    |            ^^^^^ not found in `tokio`
    |
help: consider importing one of these items
    |
38  | use crate::connection::io::sys::ext::net::sys_common::util::thread::spawn;
    |
38  | use std::thread::spawn;
    |

error[E0425]: cannot find function `spawn` in crate `tokio`
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:325:12
    |
325 |     tokio::spawn(async move { panic!(res.await);});
    |            ^^^^^ not found in `tokio`
    |
help: consider importing one of these items
    |
38  | use crate::connection::io::sys::ext::net::sys_common::util::thread::spawn;
    |
38  | use std::thread::spawn;
    |

error[E0425]: cannot find function `spawn_blocking` in module `tokio::task`
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:366:18
    |
366 |     tokio::task::spawn_blocking(move || {
    |                  ^^^^^^^^^^^^^^ not found in `tokio::task`

error[E0425]: cannot find function `spawn` in crate `tokio`
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:373:12
    |
373 |     tokio::spawn(async {
    |            ^^^^^ not found in `tokio`
    |
help: consider importing one of these items
    |
38  | use crate::connection::io::sys::ext::net::sys_common::util::thread::spawn;
    |
38  | use std::thread::spawn;
    |

error[E0698]: type inside `async fn` body must be known in this context
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:380:14
    |
380 |     for _ in 0..10 {
    |              ^ cannot infer type for type `{integer}`
    |
note: the type is part of the `async fn` body because of this `await`
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:383:9
    |
383 |         proxy.method_call::<(BigProps,), _, _, _>("com.example.dbusrs.tokiobigtest", "Ping", ()).await.unwrap();
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0698]: type inside `async fn` body must be known in this context
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:380:17
    |
380 |     for _ in 0..10 {
    |                 ^^ cannot infer type for type `{integer}`
    |
note: the type is part of the `async fn` body because of this `await`
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:383:9
    |
383 |         proxy.method_call::<(BigProps,), _, _, _>("com.example.dbusrs.tokiobigtest", "Ping", ()).await.unwrap();
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0698]: type inside `async fn` body must be known in this context
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:380:14
    |
380 |     for _ in 0..10 {
    |              ^^^^^ cannot infer type for type `{integer}`
    |
note: the type is part of the `async fn` body because of this `await`
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:383:9
    |
383 |         proxy.method_call::<(BigProps,), _, _, _>("com.example.dbusrs.tokiobigtest", "Ping", ()).await.unwrap();
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0698]: type inside `async fn` body must be known in this context
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:380:14
    |
380 |     for _ in 0..10 {
    |              ^^^^^ cannot infer type for type `{integer}`
    |
note: the type is part of the `async fn` body because of this `await`
   --> /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/dbus-tokio-0.7.0/src/connection.rs:383:9
    |
383 |         proxy.method_call::<(BigProps,), _, _, _>("com.example.dbusrs.tokiobigtest", "Ping", ()).await.unwrap();
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 11 previous errors

Some errors have detailed explanations: E0425, E0432, E0433, E0698.
For more information about an error, try `rustc --explain E0425`.
error: could not compile `dbus-tokio`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Comment by birkenfeld** [comment]
Looks like the problem is that the `macros` feature is only pulled in by dev-dependencies...

**Comment by diwic** [comment]
Fixed in 0.7.2. Thanks!
